man/systemd.netdev.xml

```diff
@@ -1979,6 +1982,94 @@
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either <literal>off</literal>, <literal>server</literal> or
+          <literal>client</literal>. A batman-adv node can either run in server mode (sharing its internet
+          connection with the mesh) or in client mode (searching for the most suitable internet connection
+          in the mesh) or having the gateway support turned off entirely (which is the default setting).
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Aggregation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables aggregation of originator messages. Defaults to
+           true.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>BridgeLoopAvoidance=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables avoidance of loops on bridges. Defaults to true.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>DistributedArpTable=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables the distributed arp table. Defaults to true.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>Fragmentation=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. Enables or disables fragmentation. Defaults to true.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>HopPenalty=</varname></term>
+        <listitem>
+          <para>The hop penalty setting allows to modify <command>batctl</command> preference for multihop
+          routes vs. short routes. The value is applied to the TQ of each forwarded OGM, thereby
+          propagating the cost of an extra hop (the packet has to be received and retransmitted
+          which costs airtime). A higher hop penalty will make it more unlikely that other nodes
+          will choose this node as intermediate hop towards any given destination.
+          The default hop penalty of '15' is a reasonable value for most setups and probably does not need
+          to be changed. However, mobile nodes could choose a value of 255 (maximum value) to avoid being
+          chosen as a router by other nodes.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>OriginatorIntervalSec=</varname></term>
+        <listitem>
+          <para>The value specifies the interval in seconds in which batman-adv floods the
+          network with its protocol information.</para>
```

Comment:
well, if a unit of "ms" is specified, also accepts msec. Maybe say so, and add a reference to systemd.time(7)